ui/components/ui/new-network-info/new-network-info.js

```diff
@@ -63,151 +72,182 @@ export default function NewNetworkInfo() {
     !isLoading &&
     showPopup && (
       <Popover
+        title={t('switchedTo')}
+        centerTitle
         onClose={onCloseClick}
         className="new-network-info__wrapper"
+        headerProps={{ marginLeft: 6 }}
         footer={
-          <Button type="primary" onClick={onCloseClick}>
-            {t('recoveryPhraseReminderConfirm')}
-          </Button>
+          <>
+            <Button
+              variant="secondary"
+              href="https://metamask.zendesk.com/hc/en-us/articles/4404424659995"
```

Comment:
Nit: Can you move the link to `ui/helpers/constants/zendesk-url.js`?

Reply:
good call, it was already in there so I'll reuse the constant